### ISSUE TITLE
[BUG] Bump feature failure

### ISSUE THREAD

**Issue by cherpglitch** [CherpChat/IssueBoard#20]
**Describe the bug**
With the timer under "My Prompts" saying "OK", I scroll down and hit the bump button. The timer then starts counting down. I refresh the page, the timer says "OK", and the prompt still has its bump button.

**To Reproduce**
1. Go to "My Prompts"
2. Scroll down to a prompt with a bump button
3. Click on "Bump"
4. Refresh page or visit directory
5. See error

**Expected behavior**
Prompt to be bumped, timer to continue after refreshing page

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: macOS Big Sur 11.2.1
 - Browser [e.g. chrome, safari] Chrome
 - Version [e.g. 22] 88.0.4324.182

**Comment by two-keys** [comment]
Strange. What's the timestamp for this prompt? Specifically, was it posted before 09/22 (month/day format)?

Also, are you experiencing this bug with just one prompt or multiple prompts?

**Comment by cherpglitch** [comment]
Yes to both, it is affecting multiple prompts posted before 09/22. I don't have any prompts since that date to test whether it is that date specifically, though.

**Comment by two-keys** [comment]
Noted.

If you can, I would recommend attempting the following for any of the prompts posted prior to 09/22:
1. Set the prompt to 'draft' from within the edit screen, then hit submit, even if it is already displaying as a draft.
2. Set the prompt back to 'posted' from within the edit screen, then hit submit.
3. Try bumping the given prompt.
